### ISSUE TITLE
missing required fields from MAF: Hugo_Symbol

### ISSUE THREAD

**Issue by renzhonglu** [PoisonAlien/maftools#77]
Hello,
I use the github clone version of maftools, and I download MAF data from cBioPortal.  the file named "data_mutations_extended.txt"  in the **.tar.gz file is a MAF format file. When I  am trying to load this maf file, I get an error message:
```
reading maf..
Error in validateMaf(maf = maf, isTCGA = isTCGA, rdup = removeDuplicatedVariants,  : 
  missing required fields from MAF: Hugo_Symbol
```
The part of headers in this MAF file are listed here:

```
- #version 2.4
- Hugo_Symbol	Entrez_Gene_Id	Center	NCBI_Build	Chromosome	Start_Position	End_Position	Strand	Variant_Classification	Variant_Type	Reference_Allele	Tumor_Seq_Allele1	Tumor_Seq_Allele2	dbSNP_RS	dbSNP_Val_Status	Tumor_Sample_Barcode	Matched_Norm_Sample_Barcode	Match_Norm_Seq_Allele1	Match_Norm_Seq_Allele2	Tumor_Validation_Allele1	Tumor_Validation_Allele2	Match_Norm_Validation_Allele1	Match_Norm_Validation_Allele2	Verification_Status	
```
Here is Hugo_Symbol header in this file, but I don't know what's causing this issue, and how can I resolve it.
Thanks!

**Comment by renzhonglu** [comment]
Hello, 
I figure out this problem, because the headers have 117 columns in this file, but the data have 119 columns. May because that two more tab-delimiters in the end of each lines, but the header-line does not have them.

Thanks you!

**Comment by PoisonAlien** [comment]
Hi,

It should have skipped the header line starting wth `#`, but I guess that is the issue. Can you manually remove first line from the file. 

You can do this from terminal.
```
grep -v "^#" data_mutations_extended.txt > data_mutations_extended_headerFixed.maf
```

**Comment by PoisonAlien** [comment]
Oh okay! Let me know if that fixes your problem.

**Comment by renzhonglu** [comment]
Hi，
Yes, I have  fixed this problem. I add two more headers (whatever names ) in the end of header-line to  match the number of data columns. It works! 
I also tried to remove lines with "#" or to add some "#" lines. Then I read the code of read.maf(), I found that my problem has nothing to do with these operations.
I will try to read other MAF files from cBioportal to see  whether all of MAF files they called have the same problem. If so, I think that the workflow to generate MAF files may be slightly different from TCGA's workflow, although majority headers are same to MAF files from TCGA harmonization data.

Thank you!
